### ISSUE TITLE
Unified Navigation Bar to improve UX

### ISSUE THREAD

**Issue by hebertialmeida** [FolioReader/FolioReaderKit#88]
Hey guys I want your opinion on this.

When I have first designed the Reader, the only scroll orientation was vertical, so, I designed a side menu. Now that we are changing that to have both, horizontal and vertical scrolling, this left menu don't make sense.

I have designed a new navigation bar that unifies all actions:
![screen shot 2016-07-11 at 12 48 09 am](https://cloud.githubusercontent.com/assets/103670/16719483/3336adc2-4701-11e6-8135-d900f9630a11.png)

Basically it kills the left menu and group everything on this bar.

These icons are:
1. Back (Closes the reader)
2. Table of Contents (Open the TOC)
3. Font and settings adjust
4. Highlights list
5. Text to Speech or Media Overlays
6. Share

**Comment by kjantzer** [comment]
I like this idea a lot. But what about consolidating the TOC and highlight/notes into one menu? iBooks in iOS does this. It would reduce some clutter and allow for a search icon when we implement that feature. 






On Sun, Jul 10, 2016 at 8:45 PM -0700, "Heberti Almeida" <notifications@github.com> wrote:












Hey guys I want your opinion on this.



When I have first designed the Reader, the only scroll orientation was vertical, so, I designed a side menu. Now that we are changing that to have both, horizontal and vertical scrolling, this left menu don't make sense.



I have designed a new navigation bar that unifies all actions:





Basically it kills the left menu and group everything on this bar.



These icons are:

1. Back (Closes the reader)

2. Table of Contents (Open the TOC)

3. Font and settings adjust

4. Highlights list

5. Text to Speech or Media Overlays

6. Share

**Comment by alexpopov** [comment]
I would suggest an 'X' icon instead of "Back" since we present modally. Looks good.

**Comment by empedocle** [comment]
I'd prefer a library icon over back or an X.

As far as the toolbar icons, perhaps highlights list, text to speech, and share can be combined into a pullover menu. With adding a search button, that seems about right to me and more might be a bit cluttered.

I haven't contributed anything yet but might someday plan to use part of FolioReader in a project and then would likely contribute back.

An edge swipe might still be ok for the TOC if swiping for page turns was limited to the text content area.

**Comment by hebertialmeida** [comment]
@kjantzer I have combined.
@alexpopov Yeah makes sense.
@empedocle First of all welcome! I think makes more sense to combine Highlights list and Table of Contents as iBooks does. 

So guys I have changed it a little bit:
![screen shot 2016-07-11 at 11 32 38 am](https://cloud.githubusercontent.com/assets/103670/16734259/4028f88c-475b-11e6-97d8-bf0abea74797.png)

**Comment by kjantzer** [comment]
Looks good @hebertialmeida 
Customizable icons is a good idea too.

**Comment by hebertialmeida** [comment]
Guys check out the PR #99
